README.md

```diff
@@ -79,10 +85,70 @@ const {
   positionals
 } = parseArgs({ args, options });
 console.log(values, positionals);
-// Prints: [Object: null prototype] { foo: true, bar: 'b' } []ss
+// Prints: [Object: null prototype] { foo: true, bar: 'b' } []
+```
+
+Detailed parse information is available for adding custom behaviours by specifying `tokens: true` in the configuration. The returned tokens have properties describing:
+
+* all tokens
+  * `kind`: { string } One of 'option', 'positional', or 'option-terminator'.
+  * `index`: { number } Index of element in `args` containing token.
+* option tokens
+  * `name`: { string } Long name of option.
+  * `rawName`: { string } How option used in args, like `-f` of `--foo`.
```

Comment:
> The one thing I think could be confusing is that an input of -abc would resolve to rawName: '-a' 

I think I agree with @aaronccasanova's feedback on rawName, if we could get it so that `name` was 'foo', but `rawName` was `-afb`, when you have the option configured:

```js
{
  options: {
    foo: {short: 'f'}
  }
}
```

This feels most intuitive to me.

Reply:
> `rawName` was `-afb` ... feels most intuitive to me

Short version: in which case the problem is with the name, not with the value.

Long version

That would be intuitive, but not as useful in practice. As a consumer of the `tokens`, I want to display the option in error messages related to how it was used. `-f` or `--foo` is directly useful. `-afb` not so much.

There is a long thread abut what we are including and why, see this comment in particular: https://github.com/pkgjs/parseargs/pull/129#discussion_r888641629

A name which better matches the higher level purpose and/or values (i.e. `-f` or `--foo`) is welcome. I explored some other names in: https://github.com/pkgjs/parseargs/pull/129#issuecomment-1146551524

Reply:
In defence of the current (not loved) name, @aaronccasanova did [finish with](https://github.com/pkgjs/parseargs/pull/129#issuecomment-1146652233):

> In both specs long and short options are referred to as option names to which rawName (resolving -f or --foo) seems well named and aligned with standards.